requirements/base-requirements.in

```diff
@@ -112,6 +112,7 @@ stripe
 suds-py3
 text-unidecode
 tinys3
+toposort
```

Comment:
Cool! I think I'd written an implementation of this a while ago, which it seems like we can remove now:  

https://github.com/dimagi/commcare-hq/blob/ded744696f37c4e2e673a5750849ac5b4b54a2dd/corehq/apps/hqcase/tasks.py#L99-L137

Reply:
ha, I missed this when I was looking. Assuming this PR goes through I'll do a followup to replace this method with the library.

Reply:
https://github.com/dimagi/commcare-hq/pull/30936